### PR TITLE
Install the (JACK-compatible) binaries if build

### PR DESCRIPTION
**Pull request by trebmuh**

### CODE REVIEW

**Comment by trebmuh** [comment]
@ninodewit : any issue with merging this one?

**Comment by trebmuh** [comment]
ping

**Comment by trebmuh** [comment]
yearly ping @ninodewit

File: Makefile [hunk 1]
```diff
@@ -43,11 +43,20 @@ install:
...
 
 	cp bin/*-ladspa.* $(DESTDIR)$(PREFIX)/lib/ladspa/
 	cp bin/*-vst.*    $(DESTDIR)$(PREFIX)/lib/vst/
 	cp -r bin/*.lv2   $(DESTDIR)$(PREFIX)/lib/lv2/
 
+ifeq ($(HAVE_JACK),true)
+	cp -r bin/Harmless  $(DESTDIR)$(PREFIX)/bin/
+	cp -r bin/Larynx  $(DESTDIR)$(PREFIX)/bin/
+	cp -r bin/Modulay  $(DESTDIR)$(PREFIX)/bin/
+	cp -r bin/Pitchotto  $(DESTDIR)$(PREFIX)/bin/
```
Comment (ninodewit) [inline Makefile]: - [x] please remove this line, Pitchotto was never completed and abandoned at some point.

**Review by ninodewit — CHANGES_REQUESTED** [review]
I'm sorry it took me years to respond. Not sure if I'll be pushing any new stuff here but if this PR helps you and other JACK users it's good to go other than the one line.

**Review by ninodewit — APPROVED** [review]

**Comment by trebmuh** [comment]
Thank you for the merge.

**Pull request merged**